tests/unit-tests/Queue/TenantAwareJobTest.php

```diff
@@ -61,4 +61,27 @@ public function serializes_tenant()
 
         $this->assertNotEquals($identified->fqdn, $restored->fqdn);
     }
+
+    /**
+     * @test
+     */
+    public function serializes_manual_tenant()
+    {
+        $this->app->make(CurrentHostname::class);
+
+        $manualHostname = $this->getReplicatedHostname();
```

Comment:
There's no reason to call this variable manual at all. As you're not using the identified hostname, please use `$hostname` as variable.